### ISSUE TITLE
DataFrame view renders odd characters (Python 3.4, Anaconda, Linux)

### ISSUE THREAD

**Issue by ianozsvald** [yhat/rodeo#64]
If I open a script that has valid DataFrames and run it in Rodeo, everything works. On the DataFrame tab I see my DataFrames, if I click View then I see a wide thin window (a grid-view perhaps?) with a long error string (also see screenshot):
```
[1;36m File [1;32m"[1;36m File [1;32m"[1;36m File [1;32m""[1;36m, line [1;32m1[0m [1;33m print df_medication_strengths.ix[:5000].to_html()[0m [1;37m ^[0m [1;31mSyntaxError[0m[1;31m:[0m invalid syntax
```

I'm using Linux Mint 17.1 64bit, Anaconda, Python 3.4, Pandas 0.15.2. I'm working off of Rodeo HEAD pulled just now. 

![screenshot from 2015-04-26 10 59 15](https://cloud.githubusercontent.com/assets/273210/7336716/5a1c4e60-ec03-11e4-97b3-1c4d660a17cc.png)

**Comment by glamp** [comment]
sorry looks like a python3 issue. fixed here 1ad444192df89c37c0622b31c01423369c4ad197

**Comment by ianozsvald** [comment]
Confirmed working with Python 3.4, thanks :-)
